source/cleaning_service.rb

```diff
@@ -0,0 +1,82 @@
+# astia au mai multe statii de curatare
+# un orar in care functioneaza => hash cu zilele saptamanii
+# masinile programate
+require_relative 'time_table'
+
+class CleaningService
+  # attr_accessor :station_count 
+  # attr_accessor :car_count
+  attr_accessor :time_table
+  attr_accessor :stations
+  attr_accessor :queue
+
+  def initialize()
+    @stations = nil
+    @queue = nil
+    @time_table = TimeTable.new
+  end
+
+  def to_s
+    result = "Cleaning Service\n"
+    stations.each {|station| result += station.to_s}
+    result += "Queue:\n"
+    queue.each {|car| result += car.to_s}
+    return result
+  end
+
+  def info
+    result = "Schedules:\n"
+    stations.each {|station|
```

Comment:
use` do..end` block instead of `{}` when you have multiple lines of code. `{}` is used when there is one line of code

```
stations.each do |station|
...
end
```